tools/packages.yaml

```diff
@@ -7935,7 +7935,7 @@ url: https://github.com/rakudo/rakudo/releases
 activity: medium
 ---
 kind: url
-name: ranger
+name: ranger_fm
```

Comment:
Why not call it `py3_ranger_fm` and add a fake package called `ranger` with this dependency?  We really should be consistent with our python package naming convention.  The `ranger` fake package makes it easier to find and install.

Reply:
I thought we used `py3` to notate packages where them being implemented in python is relevant, i.e. libraries and such, like `py3_retrying`, while packages that just happened to be implemented in python, like ranger and csvkit, don't have the prefix.

basically, if its a python library, add `py_3`, and if its not, don't.

Reply:
That's a new thought to me.  Maybe we should document these naming conventions?  I'm not a fan of renaming packages to longer, not so obvious names that make it more difficult to locate and install.

Reply:
Good point. I've opened https://github.com/chromebrew/chromebrew/issues/11631 to discuss this and similar matters further.